api.md

```diff
@@ -187,6 +202,16 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
 SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_exists => true);
 ```
 
+First, create a multinode with two data nodes, then convert table `conditions` to 
+a distributed hypertable with just time partitioning on column `time`, 
+finally add a space partitioning on `location` with two partitions:
+```sql
+SELECT add_data_node('data_node1', host => 'node1_host_address');
+SELECT add_data_node('data_node2', host => 'node2_host_address');
+SELECT create_distributed_hypertable('conditions', 'time');
+SELECT add_dimension('conditions', 'location', number_partitions => 2);
```

Comment:
```suggestion
SELECT add_dimension('conditions', 'location');
```

Reply:
@mfreed It doesn't work in this way. `add_dimension` in you suggestion will return an error that `number_partitions` (or time interval) must be provided.

Reply:
The default exists only for `create_distributed_hypertable`.

Reply:
Sounds good.  Curious -> is this fundamental or just an API thing we didn't get around to updating?